roles/webserver/templates/delayed_job.service.j2

```diff
@@ -0,0 +1,17 @@
+# Open Food Network delayed_job systemd service
+
+[Unit]
+Description=Open Food Network delayed_job
+
+[Service]
+Type=simple
+User=openfoodnetwork
+WorkingDirectory={{ current_path }}
+Environment=RAILS_ENV={{ rails_env }}
+SyslogIdentifier=openfoodnetwork-delayed_job
+PIDFile={{ delayed_job_pid }}
+
+ExecStart=script/delayed_job
```

Comment:
This line should contain `script/delayed_job start`. We're also missing an `ExecStop`

Reply:
https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files#unit-specific-section-directives
> *ExecStop=*: This indicates the command needed to stop the service. If this is not given, the process will be killed immediately when the service is stopped.

Do we need to run `script/delayed_job start` or can just let it get killed? I don't really know how it uses PIDs etc.
Possibly relevant: http://www.rubydoc.info/gems/delayed_job/4.1.4#Running_Jobs

Reply:
We need `start` in `ExecStart` for sure, and I would add `ExecStop` just to be consistent with DelayedJob API.

Reply:
We can consider using this under `[Service]` as well: 
```
Restart=always
RestartSec=30
```